### ISSUE TITLE
`<v:fill />` gets it trailing slash removed with `prettify:true`

### ISSUE THREAD

**Issue by hteumeuleu** [maizzle/framework#1158]
- Maizzle Version: 4.7.2
- Node.js Version: 20.9.0

I spent the last hour turning crazy not understanding why I couldn't get any VML background image to work in _The Outlooks_. But I just found the culprit!

Looks like the `prettify:true` build option (which is set by default in production, but I also happen to turn it on for local dev) removes the final trailing slash in `<v:fill />` elements. For example, with the default Maizzle starter, in `promotional.html` with a local build:

```html
<!--[if mso]>
<v:rect stroke="f" fillcolor="none" style="width: 600px" xmlns:v="urn:schemas-microsoft-com:vml">
<v:fill type="frame" src="https://images.unsplash.com/photo-1565932887479-b18108f07ffd?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1200&q=80" />
<v:textbox inset="0,0,0,0" style="mso-fit-shape-to-text: true"><div><![endif]-->
```

And here's the same email with a production build:

```html
<!--[if mso]>
<v:rect stroke="f" fillcolor="none" style="width: 600px" xmlns:v="urn:schemas-microsoft-com:vml">
<v:fill type="frame" src="https://images.unsplash.com/photo-1565932887479-b18108f07ffd?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1200&q=80">
<v:textbox inset="0,0,0,0" style="mso-fit-shape-to-text: true"><div><![endif]-->
```

See the difference? (Right at the end of the `v:fill` line: `&q=80" />` versus `&q=80">`)
This makes the fill element wrap the next texbox element, thus hiding the content inside the background image completely. 

Here's a test on Email on Acid showing the broken images: https://app.emailonacid.com/app/acidtest/XmaRmNuMpX10oQMYrpXreNkU6v2sFKkrtd7BkIhyklCL0/list

**Comment by hteumeuleu** [comment]
Found a temporary fix: adding a closing `</v:fill>` tag instead of relying on a self-closing tag.

```html
<v:fill type="frame" src="…"></v:fill>
```

**Comment by cossssmin** [comment]
Sorry about that, it's a loose replacement that we do when you have a HTML 5 doctype. 

Working on a fix for it and some other issues right now, will release a patch today. 

`prettier` has some issues that we need to fix with regex replacements, however if we need to keep on fighting it I'll consider going back to what we had in the past - a few better-indented HTML comments isn't worth the hassle tbh...

**Comment by cossssmin** [comment]
Released `v4.7.3`, should be fixed now.
